service/experiment-requests.yaml

```diff
@@ -0,0 +1,22 @@
+# Experiment requests have the following format:
+#
+# - experiment: 2020-06-08  # The name of the experiment
+#   fuzzers:  # The fuzzers to run in the experiment.
+#     - aflplusplus
+#     - libfuzzer
+#
+# The name of the experiment must begin with a date using this format:
+# YYYY-MM-DD (year-month-day). It's not important what timezone is used in
+# deciding the date or if this is a day or two off. The most important thing is
```

Comment:
We do we require names to start with dates? Asking because if the date doesn't really mean anything (eg. we won't necessary run the exp on the given date) than this might be confusing for people

Reply:
Right. The reason why I want dates is because that is the format we've been using: https://www.fuzzbench.com/reports/

Reply:
But yes there would be a problem with running experiments on the right date.

Reply:
Some alternatives to consider:
- Ask for just some short tag/identifier, e.g., 'libfuzzer-update' or 'my-fuzzer-v6.8' and we would auto-generate the experiment name from the `<data-of-run>-<tag>`, e.g., `2020-06-22-my-fuzzer-v6.8`
- Don't ask for any name, but auto-generate experiment name from the provided fuzzer list: `<DATE-OF-RUN>-<list-of-updated-fuzzers>`, e.g., `2020-06-22-afl-libfuzzer-honggfuzz`. (I imagine typically the list of updated fuzzers would be small in individual requests)